indico/modules/events/sessions/controllers/management/sessions.py

```diff
@@ -146,16 +151,30 @@ def _process_DELETE(self):
     def _process_PATCH(self):
         data = request.json
         updates = {}
-        if set(data.viewkeys()) > {'colors'}:
+        if set(data.viewkeys()) > {'colors', 'type_id'}:
```

Comment:
this isn't correct:

    >>> {'a', 'c'} > {'a', 'b'}
    False

What we actually want here is `set(data.viewkeys()) - {'colors', 'type_id'}` to fail if there is anything besides the allowed ones.

FWIW, with a single value in the setusing `>` worked, but with more than one on the right side it doesn't.